actions.js

```diff
@@ -0,0 +1,56 @@
+
+/* eslint-disable import/prefer-default-export */
+import { Toast } from 'native-base';
+import CONFIG from './config.js';
+
+const API_URL = CONFIG.ENV === 'PROD' ? CONFIG.API_URL : CONFIG.TEST_API_URL;
+
+export function constructContactItemsWithToken(token) {
+  return async (dispatch) => {
+    try {
+      dispatch({
+        type: 'SET_USER_DATA',
```

Comment:
Could we change this to something like "SET_CONTACTS_LIST"?

Reply:
we could but I realized eventually we will add more data to the user object that's why I did it this way. Let me know if that makes sense

Reply:
I think it's probably better to start off keeping the actions as specific as possible, so that the user data can be more easily broken up in the future. Since this action is only going to be dispatched when we have contact data to update, it probably makes sense to have a specific action for just that data mutation.

Reply:
I agree, fixed